HorizonEditor/src/EditorLayer.cpp

```diff
@@ -74,7 +77,7 @@ void EditorLayer::onUpdate(Hzn::TimeStep ts)
 		if (m_PlayMode)
 			EditorData::s_Scene_Active->onUpdate(ts);
 		else
-			EditorData::s_Scene_Active->onEditorUpdate(m_EditorCameraController.getCamera(), ts);
+			EditorData::s_Scene_Active->onEditorUpdate(m_EditorCameraController.getCamera(), ts, EditorData::assetManager);
```

Comment:
Is this receiving a new Camera position for each time the editor is updated?

Reply:
yes